omnibus/package-scripts/installer-rpm/posttrans

```diff
@@ -65,9 +65,8 @@ fi
 
 # Set helper rights
 chmod 750 ${HELPER}
-setcap cap_setuid+ep ${HELPER}
 
-$BOOTSTRAP_INSTALLER bootstrap --url "oci://public.ecr.aws/datadog/installer-package:7.54.0-devel.git.366.99b427c.pipeline.32143438-1"
+$BOOTSTRAP_INSTALLER bootstrap --package "datadog-installer"
```

Comment:
Could you clarify what this change is doing/why it's done?

Reply:
it allows to read from config which container registry to hit, and defaults to latest released version